Source/WebKit/Shared/LoadParameters.cpp

```diff
@@ -39,6 +39,8 @@ void LoadParameters::encode(IPC::Encoder& encoder) const
 #endif
     encoder << navigationID;
     encoder << frameIdentifier;
+    encoder << parentFrameIdentifier;
+    encoder << layerHostingContextIdentifier;
```

Comment:
These probably shouldn't be here.

Reply:
we need these when we call `createLocalSubframeHostedInAnotherProcess` in the web process. These are used so that we can create a LocalFrame in a web process which currently has a RemoteFrame.